.idea/NumRu.iml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You seem to have a bunch of stuff checked into Git that you shouldn't have: .idea and target/. You should git rm -r both of them, and then make sure that they are properly covered by .gitignore. (`/target` won't apply to `target` in a subdirectory.)